stakingModel/test/GoodMarketMaker.test.js

```diff
@@ -91,19 +175,223 @@ contract(
+      let gdSupply = reserveToken.gdSupply;
+      let error = await marketMaker.sell(
+                          cDAI.address,
+                          (gdSupply + 1).toString()
+                        ).catch(e => e);
+      expect(error.message).to.have.string(
+        "GD amount is higher than the total supply"
       );
     });
 
+    it("should set reserve ratio daily expansion", async () => {
+      let denom = new BN(1e15).toString();
+      let currentReserveRatioDailyExpansion = await marketMaker.reserveRatioDailyExpansion();
+      let encodedCall = web3.eth.abi.encodeFunctionCall({
+                          name: 'setReserveRatioDailyExpansion',
+                          type: 'function',
+                          inputs: [{
+                              type: 'uint256',
+                              name: '_nom'
+                          },{
+                              type: 'uint256',
+                              name: '_denom'
+                          }]
+                      }, ['1', denom]);
+      await avatar.genericCall(marketMaker.address, encodedCall, 0);
+      let newReserveRatioDailyExpansion = await marketMaker.reserveRatioDailyExpansion();
+      expect(newReserveRatioDailyExpansion).not.to.be.equal(currentReserveRatioDailyExpansion);
+      encodedCall = web3.eth.abi.encodeFunctionCall({
+                  name: 'setReserveRatioDailyExpansion',
+                  type: 'function',
+                  inputs: [{
+                      type: 'uint256',
+                      name: '_nom'
+                  },{
+                      type: 'uint256',
+                      name: '_denom'
+                  }]
+              }, ['999388834642296', denom]);
+      await avatar.genericCall(marketMaker.address, encodedCall, 0);
+      let reserveRatioDailyExpansion = await marketMaker.reserveRatioDailyExpansion();
+      expect(reserveRatioDailyExpansion).not.to.be.equal(currentReserveRatioDailyExpansion);
+    });
+
+    it("should be able to buy only by the owner", async () => {
+      let error = await marketMaker.setReserveRatioDailyExpansion(1, 1e15).catch(e => e);
+      expect(error.message).to.have.string("only Avatar can call this method");
+    });
+
+    it("should calculate amount of gd to mint based on incoming cDAI without effecting bonding curve price", async () => {
+      const priceBefore = await marketMaker.currentPrice(dai.address);
+      const toMint = await marketMaker.calculateMintInterest(
+        dai.address,
+        web3.utils.numberToHex(1e18), {
+          from: staker
+        }
+      );
+      const totalMinted = 1e18 / priceBefore.toNumber();
+      expect(toMint.toString()).to.be.equal((Math.floor(totalMinted * 100)).toString());
+      const priceAfter = await marketMaker.currentPrice(dai.address);
+      expect(priceBefore.toString()).to.be.equal(priceAfter.toString());
+    });
+
+    it("should not change the reserve ratio", async () => {
```

Comment:
description not enough, what should not change reserve ratio
same for next tests